spec/rapis/_files/paths.rb

```diff
@@ -0,0 +1,253 @@
+item "/" do
+  get do
+    consumes ["application/json"]
+    produces ["text/html"]
+    responses do
+      code "200" do
+        description "200 response"
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)